configure

```diff
@@ -136,6 +136,7 @@ def main():
     config.script_path            = config.source_root.path_by_appending("build.py")
     config.build_script_path      = config.source_root.path_by_appending("build.ninja")
     config.config_path            = config.source_root.path_by_appending(".configuration")
+    config.libdir                 = os.getenv("LIBDIR", "lib")
```

Comment:
I'm always a little scared about repeatable builds when the config script picks up from the build-host environment. Is there a way to do this via the config setup that we have instead?

Reply:
Sure, i can add it as a configure command line option, creating a "--libdir" switch using the parser with a default "lib" value. It will imply in a minor modification on build-script-impl at swift repository, but there is no problem since nobody is reviewing that pull request.

Reply:
Yeah, I'd prefer this approach. Thanks!